app/user.go

```diff
@@ -2275,7 +2275,7 @@ func (a *App) UpdateThreadReadForUser(userID, teamID, threadID string, timestamp
 	}
 	membership, storeErr := a.Srv().Store.Thread().MaintainMembership(userID, threadID, opts)
 	if storeErr != nil {
-		return nil, model.NewAppError("UpdateThreadReadForUser", "app.user.update_thread_read_for_user.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return nil, model.NewAppError("UpdateThreadReadForUser", "app.user.update_thread_read_for_user.app_error", nil, storeErr.Error(), http.StatusInternalServerError)
```

Comment:
@streamer45 - wondering if we can write a vet check for cases like these. I fear there are others hidden in our codebase.

Reply:
You are tempting me :stuck_out_tongue:  It would be nice for sure. I've seen a lot of these.

Reply:
+1 to this vet check